eslint-config-ringcentral/index.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
         es6: true,
         mocha: true
     },
-    extends: 'eslint:recommended',
+    extends: ['plugin:you-dont-need-lodash-underscore/all', 'eslint:recommended'],
```

Comment:
Disagree. This one is too strict and not cross browser. I don't think we have totally forbid lodash. A lot of cases when lodash functionality is better choice vs native build-ins. I guess we have to prefer native methods when possible, but do not dictate using only native methods by introducing this rule into guide.

Reply:
It's not all at all. It is just which are easy to replace by native implementation https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore/blob/master/lib/rules/rules.json